metadata/service/support.yml

```diff
@@ -0,0 +1,11 @@
+parameters:
+  watchdog:
+    _support:
+      collectd:
+        enabled: false
+      heka:
+        enabled: false
+      sensu:
+        enabled: false
+      sphinx:
+        enabled: true
```

Comment:
You don't ship sphinx support so set this to false

Reply:
Fixed...